bigbang/ietfattendance.py

```diff
@@ -66,13 +66,16 @@ def attendance_tables(mn):
         data = pd.concat(data, axis = 0)
     else:
         data = data[0]
+
+    print(data.shape)
```

Comment:
At some point, we should update this script to use logging rather than print statements, and then at the same time we can make the logging more meaningful than these undocumented outputs.

We don't need to make that change now, but might add it to the todo's on this script (a growing list) or open an issue.